packages/blocks/src/page-block/edgeless/components/component-toolbar/component-toolbar.ts

```diff
@@ -200,6 +203,11 @@ export class EdgelessComponentToolbar extends WithDisposable(LitElement) {
       : nothing;
   }
 
+  protected getPopperShow = (popperShow: boolean) => {
+    this._popperShow = popperShow;
```

Comment:
You should not set a value inside a `getSomething` method, whose semantics should be read only.

Reply:
Thank you for review my code. I will fix it quickly!